### PR TITLE
Update ids and Axios console logs

### PR DESCRIPTION
**Pull request by aleksandar-cakic**
- Removed non unique ids 
- Added labels to form elements
- Added loudly visible console logs to Axios requests
- Tests passing

### CODE REVIEW

File: client/src/components/ratingsReviews/NewReview.jsx [hunk 1]
```diff
@@ -134,7 +135,7 @@ class NewReview extends React.Component {
           [id]: e.target.value
         });
       }
-    } else if (id === 'Characteristic') {
+    } else if (name === 'Size' || name === 'Comfort' || name === 'Quality' || name === 'Length' || name === 'Fit' || name === 'Width') {
       this.characteristics[currentCharacteristic.id] = parseInt(e.target.value);
```
Comment (stevenharderjr) [inline client/src/components/ratingsReviews/NewReview.jsx:139]: I've noticed that you use `parseInt` a lot. Is that some sort of failsafe? It seems like most of the things you're using it on should already be numbers.

Comment (aleksandar-cakic) [reply client/src/components/ratingsReviews/NewReview.jsx:139]: In this case its not a fail safe, e.target.value returns a string.

Comment (stevenharderjr) [reply client/src/components/ratingsReviews/NewReview.jsx:139]: I never even realized the ids were strings :)

**Review by stevenharderjr — APPROVED** [review]

File: client/src/components/ratingsReviews/Sort.jsx [hunk 2]
```diff
@@ -7,9 +7,11 @@ const Sort = (props) => {
 
   return reviews ? (
     <div id='rr-sort'>
-      {fullReviews.length} reviews, sorted by <select id='rr-dropdown' className='rr-dropdown' onChange={handleReviewSort}>
+      <label for='rr-dropdown'></label>
+      {fullReviews.length} reviews, sorted by <select className='rr-dropdown' onChange={handleReviewSort}>
+
         {sortOptions.map((option, i) => (
```
[the remark this answers is not in the document]
Comment (aleksandar-cakic) [reply client/src/components/ratingsReviews/Sort.jsx:13]: Yes, one of the optional params is index. You could rewrite line 13 to be map(option, i, sortOptions) => (
I just prefer having array in front of map :)

**Pull request merged**